src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let personStatus;
+
+  if (isMarried) {
+    personStatus =
+      sex === 'f'
+        ? `${partnerName} is my husband`
+        : `${partnerName} is my wife`;
+  } else {
+    personStatus = 'I am not married';
+  }
+
+  return (
+    <div className="Person">
+      <div className="Person__name">My name is {name}</div>
+      {age && age !== null && <div className="Person__age">I am {age}</div>}
```

Comment:
```suggestion
      {age && <div className="Person__age">I am {age}</div>}
```
this condition would be enough